src/Resolver/ObjktResolver.ts

```diff
@@ -137,6 +137,17 @@ export class ObjktResolver {
     return ctx.objktAvailableRedeemablesLoader.load(objkt.id)
   }
 
+  @FieldResolver(returns => Number)
+  async mintedPrice(@Root() objkt: Objkt) {
+    const query = Action.createQueryBuilder("action").select()
```

Comment:
To avoid [GraphQL N+1](https://medium.com/the-marcy-lab-school/what-is-the-n-1-problem-in-graphql-dd4921cb3c1a) problem, you want to use DataLoader instead of running a query here.
See DataLoaders folder for examples, you will also need to add the type in types/RequestContext.ts and instanciate in utils/Context.ts